src/main/java/dk/kb/storage/webservice/KBOAuth2Handler.java

```diff
@@ -186,34 +186,38 @@ public void handleNoAuthorization(String endpoint, Set<String> endpointRoles, bo
             case ENABLED:
                 if (endpointRoles.contains(KBAuthorization.PUBLIC)) {
                     if (invalidToken) {
-                        log.debug("Failed verification of provided access token (reason={}) but access to " +
+                        log.info("Failed verification of provided access token (reason={}) but access to " +
                                   "endpoint '{}' granted as roles included '{}'",
                                   failedReason, endpoint, KBAuthorization.PUBLIC);
                     } else {
-                        log.debug("No Authorization defined in request but access to endpoint '{}' granted as roles " +
+                        log.info("No Authorization defined in request but access to endpoint '{}' granted as roles " +
```

Comment:
Do we want to log this on info? In production this could log quite a lot right?

Reply:
Not sure. If it happens a lot in production, I think there is something wrong with our setup.

Reply:
Yeah, I think you are right on this one!